app/src/main/res/layout/open_screen_recent_item.xml

```diff
@@ -1,53 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
+<FrameLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="horizontal"
-    android:paddingTop="20dip"
-    android:paddingBottom="20dip"
-    android:paddingLeft="10dip"
-    android:paddingRight="1dip"
     android:clickable="true"
     android:focusable="true"
     android:background="?android:attr/selectableItemBackground">
-
-    <ImageView
-        android:id="@+id/open_screen_db_icon"
-        android:layout_width="@dimen/tab_icon_size"
-        android:layout_height="@dimen/tab_icon_size"
-        android:layout_marginRight="6dip"
-        android:layout_gravity="center_vertical"
-        app:srcCompat="@drawable/database" />
     <LinearLayout
-        android:orientation="vertical"
-        android:layout_width="0dp"
-        android:layout_weight="1"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:gravity="center_vertical" >
-        <TextView
-            android:id="@+id/recent_item_filename"
-            android:layout_width="match_parent"
+        android:orientation="horizontal"
+        android:paddingTop="20dip"
+        android:paddingBottom="20dip"
+        android:paddingLeft="10dip"
+        android:paddingRight="1dip">
```

Comment:
Please use dp as unit (The same as dip but it is officially called dp). 
Also could you standardlize these values and put it in dimens.xml?

Also 1dp for the right padding seems negligible. Is it necessary?